packages/shared/src/cli/__test__/git.tag.test.ts

```diff
@@ -0,0 +1,9 @@
+import o from 'ospec';
+import { GitTag } from '../git.tag';
+
+o.spec('git.tag', () => {
+    o('format', () => {
+        o(/^v\d+\.\d+\.\d+(-\d+-g[0-9a-f]+)?$/.test(GitTag.version)).equals(true);
+        o(/^[0-9a-f]+$/.test(GitTag.hash)).equals(true);
```

Comment:
when git moves to sha2 hashes i think this is going to fail.

Reply:
It's a test so we'll know right away and it would be good to get a "heads up" about it